app/helpers/bank_accounts_helper.rb

```diff
@@ -0,0 +1,11 @@
+module BankAccountsHelper
+  def colorize(amount)
+    if amount > 0
+      AppConfig.account_colors.positive.css_class
```

Comment:
@Gataniel don't use class from config, let's think that class always match variable name (positive, negative, e.t.c)